### ISSUE TITLE
Remove: Win Sports +

### ISSUE THREAD

**Issue by freearhey** [iptv-org/database#14950]
### Channel ID (required)

WinSportsPlus.co

### Reason (required)

Duplicate

### Notes

https://iptv-org.github.io/channels/co/WinPlusFutbol

**Comment by BellezaEmporium** [comment]
In this case, isn't this a channel replacement ?

(replaced_by WinPlusFutbol.co ?)

**Comment by freearhey** [comment]
According to [this](https://logos.fandom.com/wiki/Win%2B_F%C3%BAtbol) the channel was simply renamed to `Win+ Fútbol`.

**Comment by BellezaEmporium** [comment]
Indeed, in this case the old name should be added in the alt_names tab, to avoid confusion.
